ui/desktop/app/styles/app.scss

```diff
@@ -100,15 +100,8 @@
 }
 
 .rose-header {
-  -webkit-app-region: drag;
```

Comment:
Why were these styles removed?

Reply:
Might be a leftover from an experiment. Oops. Will recheck this.

Reply:
Oh, I remember why padding was needed. Desktop electron shell needs the padding to accommodate min/max icons. Updated by reverting deletion.